http/src/main/resources/reference.conf

```diff
@@ -306,11 +306,11 @@ azure {
       {
         name = "cromwell-reader"
         kind = "ClusterIP"
+        path = "/cromwell"
```

Comment:
Health/status check assumes all cromwell services are located at `/cromwell`, rather than creating a new service type and status checker and unnecessary boilerplate, put this service at `/cromwell`. Proxy URL will look something like:
`"cromwell-reader": "https://LZ.servicebus.windows.net/<appName>-<uuid>/cromwell"`

Reply:
Nope, it will default to `/<name>` AKA `/cbas`

Reply:
If there’s no path defined in config it defaults to the name of the service.